docs/raw/attributes/attributes.md

```diff
@@ -9,7 +9,7 @@ tenant
 
 - Type: `list` of `attributes.TenantAttribute` 
 
-// description for tenant
+Custom attributes to store additional details about your tenants.
```

Comment:
Please exclude the whole file from the PR, I have a much more descriptive information waiting on a different PR.